browser/ui/content_settings/brave_autoplay_blocked_image_model.cc

```diff
@@ -18,32 +18,27 @@ BraveAutoplayBlockedImageModel::BraveAutoplayBlockedImageModel()
     : ContentSettingSimpleImageModel(ImageType::PLUGINS,
                                      CONTENT_SETTINGS_TYPE_AUTOPLAY) {}
 
-void BraveAutoplayBlockedImageModel::UpdateFromWebContents(
-    WebContents* web_contents) {
-  set_visible(false);
+bool BraveAutoplayBlockedImageModel::UpdateAndGetVisibility(WebContents* web_contents) {
   if (!web_contents)
-    return;
+    return false;
 
   TabSpecificContentSettings* content_settings =
       TabSpecificContentSettings::FromWebContents(web_contents);
   if (!content_settings)
-    return;
+    return false;
   if (!content_settings->IsContentBlocked(content_type()))
-      return;
+    return false;
 
-  set_visible(true);
   const gfx::VectorIcon* badge_id = &kBlockedBadgeIcon;
   const gfx::VectorIcon* icon = &kAutoplayStatusIcon;
   set_icon(*icon, *badge_id);
   set_explanatory_string_id(IDS_BLOCKED_AUTOPLAY_TITLE);
   set_tooltip(l10n_util::GetStringUTF16(IDS_BLOCKED_AUTOPLAY_TITLE));
+  return true;
 }
 
-ContentSettingBubbleModel*
-BraveAutoplayBlockedImageModel::CreateBubbleModelImpl(
-    ContentSettingBubbleModel::Delegate* delegate,
-    WebContents* web_contents,
-    Profile* profile) {
-  return new BraveAutoplayContentSettingBubbleModel(delegate,
-      web_contents, profile);
+std::unique_ptr<ContentSettingBubbleModel>
+BraveAutoplayBlockedImageModel::CreateBubbleModelImpl(ContentSettingBubbleModel::Delegate* delegate,
+      content::WebContents* web_contents) {
+  return std::make_unique<BraveAutoplayContentSettingBubbleModel>(delegate, web_contents);
```

Comment:
This change is simply removing profile which is no longer passed to bubble models.